mage_ai/ai/llm_pipeline_wizard.py

```diff
@@ -190,12 +191,13 @@
 class LLMPipelineWizard:
     def __init__(self):
         ai_config = AIConfig.load(config=get_repo_config().ai_config)
-        if ai_config.mode == AIMode.OPEN_AI:
+        if ENABLE_OPEN_AI and ai_config.mode == AIMode.OPEN_AI:
```

Comment:
Let's make it backward compatible. Existing users who don't have those env vars will stop being able to use the feature and think it's a bug

Reply:
I assigned default values in `server.py` file as True. So if ENV variabels are not available for existing customers or new customers not set up those ENVs, AI functions are default available. Only people expelicitly set those ENVs as false will block AI functions.